commands/git/git-clone-rep.md

```diff
@@ -0,0 +1,18 @@
+### Clone a repository
```

Comment:
I think this file name should be `git-clone-repo`. I think I missed it in my first review.

Reply:
Done!